### ISSUE TITLE
Support for multiple configuration files

### ISSUE THREAD

**Issue by mweinelt** [mweinelt/kea-exporter#2]
Users may want to split configuration into multiple files, usually one for DHCP4 and one for DHCP6. What should the CLI look like for that?

**Comment by espro** [comment]
I feel like the simpler method here would be to manually specify your socket paths for 4/6 and register the inotify on the socket files. The mtime will update on a reload which is necessary when a config file changes, and it means that in setups where the kea configuration is complicated and managed through a lot of includes (e.g., mine 😄), you catch all cases (e.g., a new configuration file is added). Instead of reading in the config files on start/inotify event, you use the built-in API query config-get on each service (i.e., dhcp4, dhcp6)

**Comment by mweinelt** [comment]
Hey, thanks for chiming in! I'm looking into this in a bit.

**Comment by mweinelt** [comment]
I currently only have a Kea 1.1.0 on Debian Stretch and it doesn't have `config-get` yet. Looking at Kea's release notes we see that the following features were added in 1.2.0:

> new commands - a number of new commands have been implemented. In particular, it is now possible to update (config-set), retrieve (config-get) or write the current configuration to disk (config-write). It is possible to list supported command (list-commands), check version (version-get) information or compilation details that cover dependencies versions (build-report) and more. Commands previously available only over command channel API are now also available over REST API. 

http://kea.isc.org/wiki/KeaReleaseNotes120

## idea

We can use `version-get` to determine if the Kea version supports `config-get`. If it does not we'll receive:

```json
{"command": "version-get"}
{ "result": 1, "text": "'version-get' command not supported." }
```

Else we should be able to use `config-get` and parse that instead.

**Comment by mweinelt** [comment]
> The mtime will update on a reload

That does not seem to hold true, I can't register any changes to the socket file, after reloading a changed configuration.

**Comment by mweinelt** [comment]
Fixed in 62de3676931ef5579fae48a3dfa3d6814f613923, which will soon be released in 0.4.0.
